address/book.go

```diff
@@ -217,10 +217,11 @@ func (b *Book) SetAddrManaged(ctx context.Context, addr *AddrWithKeyInfo,
 // then the status and transaction information is updated instead.
 func (b *Book) GetOrCreateEvent(ctx context.Context, status Status,
 	addr *AddrWithKeyInfo, walletTx *lndclient.Transaction,
-	outputIdx uint32, tapscriptSibling *chainhash.Hash) (*Event, error) {
+	outputIdx, heightHint uint32, tapscriptSibling *chainhash.Hash) (*Event, error) {
```

Comment:
Realized that `lndclient.Transaction` actually has a `BlockHeight` field, but it can be zero at times, so I guess the extra param is actually warranted.

Reply:
N/A anymore.